Java/src/main/java/com/planes/Plane.java

```diff
@@ -19,15 +19,15 @@ public String getModel() {
         return model;
     }
 
-    public int getMS() {
+    public int getMaxSpeed() {
         return maxSpeed;
     }
```

Comment:
please add Lombook usage for constructor/getters/setters creation

Reply:
resolved